app/assets/stylesheets/application.sass.scss

```diff
@@ -174,5 +174,16 @@ $color_app-pale-blue: #ccdff1;
 }
 
 .app-status {
+  align-items: center;
+  display: inline-flex;
   font-weight: bold;
+  gap: 4px;
+}
+
+.app-status--green {
+  color: #007f3b;
```

Comment:
Is there an `nhsuk-color` for this?

Reply:
Yes! Thanks, nice catch!

Commit amended.

Reply:
See: https://github.com/nhsuk/manage-childrens-vaccinations-prototype/blob/654cf2ecff49fb1acc8a704921abb119cf96c88f/app/assets/stylesheets/application.scss#L120-L137

Reply:
Amending.